<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddNewItemDialog</class>
 <widget class="QDialog" name="AddNewItemDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>310</width>
    <height>504</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add new items</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>&amp;Type</string>
     </property>
     <property name="buddy">
      <cstring>input_type_comboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QComboBox" name="input_type_comboBox">
     <property name="sizeAdjustPolicy">
      <enum>QComboBox::AdjustToContents</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>&amp;Size</string>
     </property>
     <property name="buddy">
      <cstring>input_size_lineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QLineEdit" name="input_size_lineEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>&amp;Mark</string>
     </property>
     <property name="buddy">
      <cstring>input_mark_comboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="2">
    <widget class="QComboBox" name="input_mark_comboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>M&amp;odel</string>
     </property>
     <property name="buddy">
      <cstring>input_model_comboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="2">
    <widget class="QComboBox" name="input_model_comboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="2">
    <widget class="QLabel" name="label_9">
     <property name="text">
      <string>&amp;Rentalgroup</string>
     </property>
     <property name="buddy">
      <cstring>input_rentalgroup_comboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="2">
    <widget class="QComboBox" name="input_rentalgroup_comboBox">
     <property name="editable">
      <bool>false</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>&amp;Year</string>
     </property>
     <property name="buddy">
      <cstring>input_year_lineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="5" column="2">
    <widget class="QLineEdit" name="input_year_lineEdit"/>
   </item>
   <item row="6" column="0" colspan="2">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>&amp;Condition</string>
     </property>
     <property name="buddy">
      <cstring>input_condition_comboBox</cstring>
     </property>
    </widget>
   </item>
   <item row="6" column="2">
    <widget class="QComboBox" name="input_condition_comboBox">
     <property name="editable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QLabel" name="label_8">
     <property name="text">
      <string>&amp;Price</string>
     </property>
     <property name="buddy">
      <cstring>input_price_lineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="7" column="2">
    <widget class="QLineEdit" name="input_price_lineEdit"/>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label_10">
     <property name="text">
      <string>&amp;Note</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <property name="buddy">
      <cstring>input_note_textEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="8" column="2">
    <widget class="QTextEdit" name="input_note_textEdit">
     <property name="tabChangesFocus">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>&amp;Id</string>
     </property>
     <property name="buddy">
      <cstring>input_id_lineEdit</cstring>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QLineEdit" name="input_id_lineEdit"/>
   </item>
   <item row="10" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>23</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="input_add_pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Add</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="input_undo_pushButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>&amp;Undo</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="input_done_pushButton">
       <property name="text">
        <string>&amp;Done</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="11" column="0" colspan="3">
    <widget class="QLabel" name="label_11">
     <property name="text">
      <string>Log of additions</string>
     </property>
    </widget>
   </item>
   <item row="12" column="0" colspan="3">
    <widget class="QTextEdit" name="output_log_textEdit">
     <property name="readOnly">
      <bool>true</bool>
     </property>
     <property name="html">
      <string>&lt;!DOCTYPE HTML PUBLIC &quot;-//W3C//DTD HTML 4.0//EN&quot; &quot;http://www.w3.org/TR/REC-html40/strict.dtd&quot;&gt;
&lt;html&gt;&lt;head&gt;&lt;meta name=&quot;qrichtext&quot; content=&quot;1&quot; /&gt;&lt;style type=&quot;text/css&quot;&gt;
p, li { white-space: pre-wrap; }
&lt;/style&gt;&lt;/head&gt;&lt;body style=&quot; font-family:'Sans Serif'; font-size:7pt; font-weight:400; font-style:normal;&quot;&gt;
&lt;table border=&quot;0&quot; style=&quot;-qt-table-type: root; margin-top:4px; margin-bottom:4px; margin-left:4px; margin-right:4px;&quot;&gt;
&lt;tr&gt;
&lt;td style=&quot;border: none;&quot;&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;
&lt;p style=&quot;-qt-paragraph-type:empty; margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;&quot;&gt;&lt;/p&gt;&lt;/td&gt;&lt;/tr&gt;&lt;/table&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>input_type_comboBox</tabstop>
  <tabstop>input_size_lineEdit</tabstop>
  <tabstop>input_mark_comboBox</tabstop>
  <tabstop>input_model_comboBox</tabstop>
  <tabstop>input_rentalgroup_comboBox</tabstop>
  <tabstop>input_year_lineEdit</tabstop>
  <tabstop>input_condition_comboBox</tabstop>
  <tabstop>input_price_lineEdit</tabstop>
  <tabstop>input_note_textEdit</tabstop>
  <tabstop>input_id_lineEdit</tabstop>
  <tabstop>input_add_pushButton</tabstop>
  <tabstop>input_done_pushButton</tabstop>
  <tabstop>output_log_textEdit</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
